From effb39a5b9a0a7e320d8a2a79d4ad9a74b8c8d36 Mon Sep 17 00:00:00 2001
From: INTEL-CAMERA WIP <wip@localhost>
Date: Tue, 11 Aug 2026 14:24:50 +0200
Subject: [PATCH 5/5] BOZZA media: ipu-bridge: Add GalaxyCore GC5035 and GC8034

NON INVIABILE: le link frequency non sono confermate su questa
macchina. Vengono dalla patch Intel per Alder Lake-M (GC5035) e dal BSP
Rockchip (GC8034), entrambe piattaforme diverse. Manca il Signed-off-by.

Verificato che l'SSDB di questa macchina NON contiene la link
frequency: il campo maxlanespeed non viene popolato dal BIOS e resta a
zero, coerentemente con il commento in testa alla tabella.
---
 drivers/media/pci/intel/ipu-bridge.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/media/pci/intel/ipu-bridge.c b/drivers/media/pci/intel/ipu-bridge.c
index 88581a4c0..6ac1bb9e9 100644
--- a/drivers/media/pci/intel/ipu-bridge.c
+++ b/drivers/media/pci/intel/ipu-bridge.c
@@ -49,6 +49,10 @@
  * Please keep the list sorted by ACPI HID.
  */
 static const struct ipu_sensor_config ipu_supported_sensors[] = {
+	/* GalaxyCore GC5035 */
+	IPU_SENSOR_CONFIG("GCTI5035", 1, 438000000),
+	/* GalaxyCore GC8034 */
+	IPU_SENSOR_CONFIG("GCTI8034", 1, 336000000),
 	/* Himax HM11B1 */
 	IPU_SENSOR_CONFIG("HIMX11B1", 1, 384000000),
 	/* Himax HM2170 */
-- 
2.47.3

